projects/ngx-datatable/src/lib/utils/math.ts

```diff
@@ -75,6 +75,27 @@ function scaleColumns(colsByGroup: any, maxWidth: any, totalFlexGrow: any) {
       }
     }
   } while (remainingWidth !== 0);
+
+  // Adjust for any remaining offset in computed widths vs maxWidth by tweaking the biggest column
+  let totalWidthAchieved = 0;
+  const biggestColumnRef = {
+    width: 0,
+    ref: null
+  };
+
+  for (const attr in colsByGroup) {
+    if (colsByGroup.hasOwnProperty(attr)) {
+      for (const column of colsByGroup[attr]) {
+        totalWidthAchieved += column.width;
+
+        if (column.width > biggestColumnRef.width) {
+          biggestColumnRef.ref = column
+        }
+      }
+    }
+  }
+  const remainingDelta = maxWidth - totalWidthAchieved;
+  biggestColumnRef.ref.width += remainingDelta;
```

Comment:
` biggestColumnRef.ref` could potentially be `null`, so I guess we want to put an `if` around:

```suggestion
  if (biggestColumnRef.ref) {
    biggestColumnRef.ref.width += maxWidth - totalWidthAchieved;
  }
```

Reply:
Fixed